### PR TITLE
Added padding in the headline on service-mesh-landscape

### LINKED ISSUE #3471: No padding in the headline on page: https://layer5.io/service-mesh-landscape
#### Description

No padding in the headline on page: layer5.io/service-mesh-landscape from 

#### Page Link:

[Link](https://layer5.io/service-mesh-landscape)

#### Screenshots

Before 
image

After:

https://user-images.githubusercontent.com/67385503/203110793-60118215-4a52-48f9-9520-3e1f43694055.mp4

#### Environment:
- Host OS: 
- Browser: 

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by gogoiakash2311**
Signed-off-by: Akash Gogoi <gogoiakash09@gmail.com>

**Description**

This PR fixes #3471

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b521cb490b8be275a30be60747b673f4a90b7c91 at: https://638d593f0f57c32b89b0a56a--layer5.netlify.app

File: src/sections/Landscape/LandscapeGrid.style.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const LandscapePageWrapper = styled.div`
 		color:${props => props.theme.text};
 	}
 	#service-mesh-comparison-strength {
-            margin: -2.5rem auto 6rem;
+            margin: 0.5rem 1.2rem 5rem;
```
Comment (Nikhil-Ladha) [inline src/sections/Landscape/LandscapeGrid.style.js:14]: Only update the `left, right` margin no need to update the `top, bottom` margin, i.e, `margin: -2.5rem 1rem 5rem` should be fine.

Comment (gogoiakash2311) [reply src/sections/Landscape/LandscapeGrid.style.js:14]: Okay I'll update it

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 2cf786700e61aab1e40ec30fbbee02d5f5a8a47a at: https://6392eaf052c765021802b089--layer5.netlify.app

**Pull request merged**